app/views/links/edit.html.erb

```diff
@@ -25,7 +25,7 @@
     </div>
 
     <div class='actions add-vertical-margins col-xs-6 no-gutter'>
-      <%= link_to 'Cancel', local_authority_with_service_path(local_authority_slug: @local_authority.slug, service_slug: @service.slug), class: 'btn btn-default add-right-margin' %>
+      <%= link_to 'Cancel', flash[:back], class: 'btn btn-default add-right-margin' %>
```

Comment:
instead of using `flash[:back]` here, would it be possible to use a helper method to wrap this functionality, and provide a fallback?